ql/src/java/org/apache/hadoop/hive/ql/exec/ReplCopyTask.java

```diff
@@ -159,7 +159,7 @@ protected int execute(DriverContext driverContext) {
         if (!rwork.getListFilesOnOutputBehaviour(oneSrc)){
```

Comment:
at the end of large database design this class will be removed. For now not removing partial code unless the IDE explicitly states the same as its just going to be unnecessary since we can remove the whole class towards the end.